### DISCUSSION TITLE
Lists disappear on Android

### DISCUSSION THREAD

**Discussion by paustint** [community/community#63993] [Other Feature Feedback, Questions, & Ideas]
I created a new list and it didn't show up in the lists as a selectable option after creation.
I tried to create again and it said the name was taken.

Later in the day I went to add another repo to some lists. I clicked the first list checkbox then when I clicked a second list checkbox the first list I selected disappeared completely.

I have a decent number of lists (maybe 12 or 16) and a few have emoji, but nothing else special AFAIK.

EDIT: I just figured out the issue is related to scrolling.
1. The nested scrolling (drawer scrolls and list scrolls) is a bit finicky and takes a few tries to complete the action
2. There is no visual indication that I have scrolled down the list
3. a newly added item was placed outside the current visible region, and I thought it just hadn't gotten created

**Comment by techexpert0119** [comment]
Based on the additional information you provided in your edit, it seems that the problem is related to scrolling. The nested scrolling behavior, where the drawer and list both have scrolling capabilities, can be a bit finicky and require multiple attempts to complete an action successfully. Furthermore, there is no visual indication that you have scrolled down the list, and it's possible that the newly added item was placed outside the current visible region, leading you to believe that it hadn't been created.

To address these issues, I suggest trying the following troubleshooting steps:

When creating a new list, ensure that you're not inadvertently using a name that is already in use by another list. Try using a unique name, or consider modifying the name slightly to differentiate it from existing lists.

To avoid the disappearance of previously selected lists, try scrolling down the list of available lists before selecting additional checkboxes. This will ensure that all the options remain visible and prevent any unintentional deselection.

Pay close attention to the scrolling behavior while creating or modifying lists. Even though there may not be a visual indication of scrolling, be aware that the list might extend beyond the visible region. Take your time and carefully check if the item has been created and placed correctly within the list.

If the issues persist despite these steps, it would be helpful to reach out to GitHub support for further assistance. They can investigate the problem more thoroughly and provide specific guidance based on your account settings and the platform's current behavior.

I hope these suggestions help you overcome the difficulties you've been experiencing. If you have any further questions or need additional support, please don't hesitate to ask.

**Comment by DRAGONOSHKA** [comment]
Кто где

**Comment by LinukaAr** [comment]
It seems like you're encountering some usability issues and inconsistencies in the interface you're using, particularly related to creating and managing lists, as well as scrolling behavior. It's not uncommon for software to have occasional glitches or usability challenges. Based on the information you've provided, here are some general suggestions to help you navigate and potentially mitigate these issues:

Refresh the Interface: Sometimes, a simple refresh of the interface or a restart of the application can help resolve temporary display or interaction issues. If you encounter unexpected behavior, try refreshing the page or restarting the application.

Clear Cache and Cookies: Cached data can sometimes cause display issues. Try clearing your browser's cache and cookies if you're using a web application. This might help in ensuring that you're seeing the most up-to-date information.

Report the Issue: If you believe the issues you're facing are due to a bug or usability problem in the software, consider reporting the problem to the software's support team or developers. They might be able to provide you with more specific guidance or release updates to address the issues.

Try Different Browsers or Devices: Sometimes, display or interaction issues can be browser-specific. If you're using a web application, try accessing it using a different browser or device to see if the issues persist.

Check for Updates: Ensure that you're using the latest version of the software. Updates often include bug fixes and improvements that can address issues you might be encountering.

Review User Documentation: If the software has user documentation or a help section, take some time to review it. It might contain information about known issues, workarounds, or instructions for using certain features.

Adjust Scrolling Behavior: Since you've identified that the issue is related to scrolling behavior, try scrolling more deliberately and slowly to ensure that you're able to see all parts of the interface, including newly added items.

Provide Feedback: If the software allows you to provide feedback or suggestions, consider sharing your experience with the development team. Constructive feedback can help them improve the user experience.

Remember that software interfaces can vary widely, and these suggestions are general in nature. The best course of action might depend on the specific software you're using and the nature of the issues you're encountering. If possible, refer to the software's official documentation or support channels for more tailored assistance.
